azafea/event_processors/endless/metrics/tests_v2/test_events.py

```diff
@@ -650,6 +650,42 @@ def test_hack_clubhouse_progress_event_with_unknown_key():
     assert not hasattr(event, 'unknown')
 
 
+def test_valid_linux_package_opened():
+    from azafea.event_processors.endless.metrics.v2.model import LinuxPackageOpened
+
+    payload = GLib.Variant('mv', GLib.Variant('s', 'gnome-calendar.deb'))
+    lpo = LinuxPackageOpened(payload)
+    assert lpo.argv == ['gnome-calendar.deb']
```

Comment:
Could you add a second test here that creates the payload in the expected `as` format to ensure it gets through the parsing correctly?